Failed tests

When I run tests with pylabrad, I get numerous failures:
```
============================= test session starts =============================
platform win32 -- Python 2.7.9 -- py-1.4.26 -- pytest-2.6.4
collected 38 items

labrad/test/test_backends.py .....
labrad/test/test_client.py FFFFFF
labrad/test/test_refresh.py F
labrad/test/test_types.py .F............
labrad/test/test_units.py ...........x

================================== FAILURES ===================================
_______________________ ClientTests.testCompoundPacket ________________________

self = <labrad.test.test_client.ClientTests testMethod=testCompoundPacket>

    def testCompoundPacket(self):
>       pts = self._get_tester()

labrad\test\test_client.py:60:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
labrad\test\test_client.py:20: in _get_tester
    self.assertTrue(hasattr(self.cxn, 'python_test_server'))
E   AssertionError: False is not true
_________________________ ClientTests.testConnection __________________________

self = <labrad.test.test_client.ClientTests testMethod=testConnection>

    def testConnection(self):
        servers = self.cxn.servers
        self.assertTrue(len(servers.keys()) > 0)
        self.assertTrue('manager' in servers)
>       self.assertTrue('python_test_server' in servers)
E       AssertionError: False is not true

labrad\test\test_client.py:27: AssertionError
_______________________ ClientTests.testContextWrappers _______________________

self = <labrad.test.test_client.ClientTests testMethod=testContextWrappers>

    def testContextWrappers(self):
        cxn1 = self.cxn()
        cxn2 = self.cxn()

>       pts1 = cxn1.python_test_server

labrad\test\test_client.py:119:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
labrad\client.py:228: in __getattr__
    return self._attrs[key]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = anritsu_server
cryo_notifier
dac_calibration
data_vault
dr_logger
ghz_fpgas
gp...server
telecomm_server
vince_direct_ethernet
vince_gpib_bus
vince_serial_server
key = 'python_test_server'

    def __getitem__(self, key):
        try:
            return super(DynamicAttrDict, self).__getitem__(key)
        except KeyError:
            # force refresh and try again
            if self._parent:
                self._parent.refresh(now=True)
            try:
                return super(DynamicAttrDict, self).__getitem__(key)
            except KeyError:
>               raise NotFoundError(key)
E               NotFoundError: (10) Could not find "python_test_server". [payloa
d=None]

labrad\client.py:129: NotFoundError
_________________________ ClientTests.testExceptions __________________________

self = <labrad.test.test_client.ClientTests testMethod=testExceptions>

    def testExceptions(self):
>       pts = self._get_tester()

labrad\test\test_client.py:106:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
labrad\test\test_client.py:20: in _get_tester
    self.assertTrue(hasattr(self.cxn, 'python_test_server'))
E   AssertionError: False is not true
___________________________ ClientTests.testServer ____________________________

self = <labrad.test.test_client.ClientTests testMethod=testServer>

    def testServer(self):
>       pts = self._get_tester()

labrad\test\test_client.py:36:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
labrad\test\test_client.py:20: in _get_tester
    self.assertTrue(hasattr(self.cxn, 'python_test_server'))
E   AssertionError: False is not true
__________________________ ClientTests.testTupleKeys __________________________

self = <labrad.test.test_client.ClientTests testMethod=testTupleKeys>

    def testTupleKeys(self):
        # allow the use of tuples as packet keys
>       pts = self._get_tester()

labrad\test\test_client.py:93:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
labrad\test\test_client.py:20: in _get_tester
    self.assertTrue(hasattr(self.cxn, 'python_test_server'))
E   AssertionError: False is not true
________________________________ test_refresh _________________________________

    def test_refresh():
        with labrad.connect() as cxn:

            assert not hasattr(cxn, 'refresh_test_server')

            with util.syncRunServer(RefreshServer1()):
                cxn.refresh()
                assert hasattr(cxn, 'refresh_test_server')
                rts = cxn.refresh_test_server

                assert hasattr(rts, 'greet')
>               assert rts.greet.accepts == ['_']
E               assert [''] == ['_']
E                 At index 0 diff: '' != '_'
E                 Use -v to get the full diff

labrad\test\test_refresh.py:43: AssertionError
___________________ LabradTypesTests.testDefaultFlatAndBack ___________________

self = <labrad.test.test_types.LabradTypesTests testMethod=testDefaultFlatAndBac
k>

    def testDefaultFlatAndBack(self):
        """
            Test roundtrip python->LabRAD->python conversion.

            No type requirements are given in these tests. In other words, we al
low
            pylabrad to choose a default type for flattening.

            In this test, we expect A == unflatten(*flatten(A)). In other words,

            we expect the default type chosen for each object to unflatten as
            an object equal to the one originally flattened.
            """
        tests = [
            # simple types
            None,
            True, False,
            1, -1, 2, -2, 2**31-1, -2**31,
            1L, 2L, 3L, 4L, 0L, 0xFFFFFFFFL,
            '', 'a', '\x00\x01\x02\x03',
            datetime.now(),

            # values
            5.0,
            Value(6, ''),
            Value(7, 'ms'),
            8+0j,
            Complex(9+0j, ''),
            Complex(10+0j, 'GHz'),

            # ValueArray and ndarray
            # These types should be invariant under flattening followed by
            # unflattening. Note, however, that since eg. [1, 2, 3] will
            # unflatten as ndarray with dtype=int32, we do not put lists
            # in this test.
            U.ValueArray([1, 2, 3], 'm'),
            U.ValueArray([1j, 2j, 3j], 's'),
            np.array([1, 3, 4], dtype='int32'),
            np.array([1.1, 2.2, 3.3]),

            # clusters
            (1, True, 'a'),
            ((1, 2), ('a', False)),

            # lists
            [],
            #[1, 2, 3, 4],
            #[1L, 2L, 3L, 4L],
            [[]],
            [['a', 'bb', 'ccc'], ['dddd', 'eeeee', 'ffffff']],

            # more complex stuff
            [(1L, 'a'), (2L, 'b')],
        ]
        for data_in in tests:
>           data_out = T.unflatten(*T.flatten(data_in))

labrad\test\test_types.py:150:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
labrad\types.py:340: in flatten
    s, t = _flatten_to(obj, types, endianness)
labrad\types.py:356: in _flatten_to
    s, t = t.__flatten__(obj, endianness)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = LRWord(), n = -2147483648L, endianness = '>'

    def __flatten__(self, n, endianness):
        if not isinstance(n, (int, long, np.integer)):
            raise FlatteningError(n, self)
        if n > 0xFFFFFFFF or n < 0:
>           raise ValueError("out of range for type w: {0}".format(n))
E           ValueError: out of range for type w: -2147483648

labrad\types.py:604: ValueError
========= 8 failed, 29 passed, 1 xfailed, 1 warnings in 2.23 seconds ==========
```
Is this expected?

--- a/labrad/test/test_types.py
+++ b/labrad/test/test_types.py
@@ -109,7 +109,7 @@ class LabradTypesTests(unittest.TestCase):
             # simple types
             None,
             True, False,
-            1, -1, 2, -2, 2**31-1, -2**31,
+            1, -1, 2, -2, 0x7FFFFFFF, -0x80000000,
             1L, 2L, 3L, 4L, 0L, 0xFFFFFFFFL,
             '', 'a', '\x00\x01\x02\x03',
             datetime.now(),

